lib/controllers/articleCtrl.js

```diff
@@ -84,6 +92,7 @@ exports.byVanity = function (req, res, next) {
 							image: imageHelper
 						},
 						useCoralTalk,
+						commentsMigrationMessage,
```

Comment:
I am not a huge fan of having `let` set and altering them along the way, it means you need to mentally track what its value is through the flow of the code. Its not a big deal in this case because this is going to be short lived and it fairly self contained but I still think something like this would be nicer.

```suggestion
						commentsMigrationMessage: now >= startOf2020 ? true : false
```

Reply:
Done